octue/log_handlers.py

```diff
@@ -73,14 +73,14 @@ def get_remote_handler(logger_uri, formatter=None):
     return handler
 
 
-def get_formatter():
-    """Get the correct formatter for the environment. If the environment is Google Cloud Run, use a log handler with a
-    formatter that doesn't include the timestamp in the log message context to avoid the date appearing twice in the
-    Google Cloud Run logs (Google adds its own timestamp to log messages).
+def get_metadata_schema_for_current_compute_platform():
+    """Get the correct metadata schema for the environment. If the environment is Google Cloud Run, use a log handler
+    with a formatter that doesn't include the timestamp in the log message context to avoid the date appearing twice in
+    the Google Cloud Run logs (Google adds its own timestamp to log messages).
```

Comment:
The name `get_metadata_schema_for_current_compute_platform` is very confusing, because:

- so much work we do is around helping people handle metadata and schema. Stick to language synonymous with the python logging module. Perhaps `get_formatter_elements()`?
- it's weirdly long and specific to include `for_current_compute_platform`, when there aren't any other options there as other functions.